dev/io.openliberty.io.opentelemetry.1.29/bnd.bnd

```diff
@@ -15,32 +15,32 @@ bVersion=1.0
 openTelemetryVersion=1.29.0
 alphaOpenTelemetryVersion=1.29.0.alpha
 
-src: src, resources
-
-Bundle-Name: io.openliberty.io.opentelemetry.1.1
-Bundle-SymbolicName: io.openliberty.io.opentelemetry.1.1; singleton:=true
-Bundle-Description: MicroProfile Telemetry Internal packages
-
 javac.source: 11
 javac.target: 11
 
+src: src, resources
+
+Bundle-Name: io.openliberty.io.opentelemetry.1.29
+Bundle-SymbolicName: io.openliberty.io.opentelemetry.1.29; singleton:=true
+Bundle-Description: OpenTelemetry bundle
+
 Import-Package: \
-  javax.net,\
-  javax.net.ssl,\
-  okhttp3,\
-  okio,\
-  zipkin2,\
-  zipkin2.codec,\
-  zipkin2.internal,\
-  zipkin2.reporter,\
-  zipkin2.reporter.okhttp3,\
-  jakarta.interceptor,\
-  jakarta.enterprise.util,\
-  com.ibm.wsspi.classloading,\
-  org.osgi.framework,\
-  com.ibm.websphere.ras,\
-  com.ibm.websphere.ras.annotation,\
-  io.openliberty.microprofile.telemetry.common.resources
+	javax.net,\
+	javax.net.ssl,\
+	okhttp3,\
+	okio,\
+	zipkin2,\
+	zipkin2.codec,\
+	zipkin2.internal,\
+	zipkin2.reporter,\
+	zipkin2.reporter.okhttp3,\
+	jakarta.interceptor,\
+	jakarta.enterprise.util,\
+	com.ibm.wsspi.classloading,\
+	org.osgi.framework,\
+	com.ibm.websphere.ras,\
+	com.ibm.websphere.ras.annotation,\
+	io.openliberty.microprofile.telemetry.internal.common.resources
```

Comment:
We removed the bundle dependency on common so we should probably remove the import too